assets/scripts/util/street_analytics.js

```diff
@@ -47,18 +47,18 @@ const getAnalyticsFromStreet = (street, locale) => {
   return { segments, street }
 }
 
-const NO_CAPACITY = { average: 0, potential: 0 }
-const UNDEFINED_CAPACITY = { average: undefined, potential: undefined }
+const NO_CAPACITY = { average: 0, potential: 0, display: true }
+const UNDEFINED_CAPACITY = { average: 0, potential: 0, display: false }
 
 const CAPACITIES = {
-  sidewalk: { average: 19000, potential: 19000 },
-  'drive-lane': { average: 1500, potential: 2000 },
-  'bike-lane': { average: 14000, potential: 14000 },
-  scooter: { average: 14000, potential: 14000 },
-  'light-rail': { average: 18000, potential: 20000 },
-  streetcar: { average: 18000, potential: 20000 },
-  'bus-lane': { average: 5000, potential: 8000 },
-  'magic-carpet': { average: 2, potential: 3 }
+  sidewalk: { average: 19000, potential: 19000, display: true },
+  'drive-lane': { average: 1500, potential: 2000, display: true },
+  'bike-lane': { average: 14000, potential: 14000, display: true },
+  scooter: { average: 14000, potential: 14000, display: true },
+  'light-rail': { average: 18000, potential: 20000, display: true },
+  streetcar: { average: 18000, potential: 20000, display: true },
+  'bus-lane': { average: 5000, potential: 8000, display: true },
+  'magic-carpet': { average: 2, potential: 3, display: true }
```

Comment:
I have a thought about the `display` property: it's basically always true, unless, in the special case of `UNDEFINED_CAPACITY`, it's false. So, what if instead of making it a required property, we assume its default value is `true`?

The lookup in `Segment.jsx` can destructure to assume display is `true` unless explicitly set to `false`.